core/assets/bundles/bundle_de.properties

```
@@ -1459,7 +1460,7 @@ block.ripple.description = Schießt mehrere Schüsse gleichzeitig auf weit entfe
 block.cyclone.description = Schießt explodierende Geschosse auf Gegner.
 block.spectre.description = Schießt große, panzerbrechende Kugeln auf Luft- und Bodenziele.
 block.meltdown.description = Lädt sich auf und schießt einen starken, durchgängigen Laser auf Gegner. Braucht Kühlung.
-block.foreshadow.description = Schießt einen einzigen Schuss mit einem einzigen Ziel über eine erstaunlich große Distanz.
+block.foreshadow.description = Schießt einen einzigen Schuss mit einem einzigen Ziel über eine erstaunlich große Distanz. Zielt auf Gegner mit mehr Leben.
```

```suggestion
block.foreshadow.description = Schießt einen einzigen, starken Schuss mit einem einzigen Ziel über massive Distanzen. Zielt auf Gegner mit mehr maximalen Lebenspunkten.
```
Obwohl ich es schwachsinnig finde, ham wir HP als Lebenspunkte durchgesetzt, wir sollten das auch hier tun.
Könnte diskutieren ob wir statt schießt 'feuert' nehmen, wenn das weiterhin nach ner elektromagnetischer Railgun designed bleibt, katapultiert sich das projectil eher

---

ich schließe mich Shawak an: "massive Distanzen" klingt irgendwie falsch; dass der Schuss stark ist ist irgendwie klar, das steht auch nicht im Englischen drinne; bei "maximalen Lebenspunkten" bin ich mir noch unsicher, was ist deine Meinung dazu @Shawak ?

---

"Zielt auf Gegner mit den meisten Lebenspunkten." finde ich aussagekräftiger. "maximalen Lebenspunkten" klingt so, als würden damit Gegner gemeint sein, deren Lebenspunkte prozentual höher sind.